### PR TITLE
Split delete_existing_responses into multiple functions

### PR DESCRIPTION
**Pull request by nnhimes**
Please set the title of your pull request in the following format: `[Project id/Independent study/Issue fix/...]. project name`, eg., `E904. Integrating courses and assignments on Expertiza with Moodle`

We suggest you including the below information in your pull request:
- A description of the changes proposed in the pull request.
- @mentions of the person or team responsible for reviewing proposed changes.
---
About Expertiza Bot
- If you have any questions about comments given by the expertiza-bot [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762412918), you could create a comment in your pull request with the format `/dispute [UUID1] [UUID2]` [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762430057) then the professor and TAs will be notified.
- [Here is a video](https://tinyurl.com/internet-bots) about how to communicate to expertiza-bot.

### CODE REVIEW

**Comment by nnhimes** [comment]
Created two new functions to gather the info needed before the delete_existing_responses function actually deletes existing responses.

**Comment by nnhimes** [comment]
The new functions also need to have rspec tests

**Review by suryamak — APPROVED** [review]
nice

**Comment by nnhimes** [comment]
Lets include the rspec tests in this PR before merging

File: spec/helpers/answer_helper_spec.rb [hunk 1]
```diff
@@ -38,6 +38,18 @@
...
+    it 'logs the response_id if in active period for each of the questions answers' do
+
+    end
+  end
+
+  describe '#log_response_info' do
+    it 'logs info from each response_id to be used in answer deletion' do
+
+    end
+  end
```
Comment (BoergerJ) [inline spec/helpers/answer_helper_spec.rb:51]: I assume these are going to be filled out at some point?

Comment (nnhimes) [reply spec/helpers/answer_helper_spec.rb:51]: Yeah I'm trying right now lol

**Review by BoergerJ — APPROVED** [review]

**Pull request merged**